code/API_definitions/qos-profiles.yaml

```diff
@@ -124,20 +128,20 @@ paths:
         "503":
           $ref: "#/components/responses/Generic503"
 
-  /qos-profiles/{name}:
+  /retrieve-qos-profile/{name}:
     get:
       tags:
         - QoS Profiles
       summary: "Get QoS Profile for a given name"
-      operationId: getQosProfile
+      operationId: retrieveQosProfile
       description: |
         Returns a QoS Profile that matches the given name.
 
         The access token may be either a 2-legged or 3-legged access token. If the access token is 3-legged, a QoS Profile is only returned if available to all end users associated with the access token.
 
       security:
         - openId:
-            - qos-profiles:qos-profiles:read
+            - qos-profiles:retrieve-qos-profiles
```

Comment:
I don't see the need to have different scopes for the two endpoints. I'd suggest this endpoint also has scope `qos-profiles:read`.